js/main.js

```diff
@@ -25,3 +28,76 @@ function handleSubmit(event) {
   $image.setAttribute('src', 'images/placeholder-image-square.jpg');
   $formEntry.reset();
 }
+
+function renderEntry(data) {
+  var liRow = document.createElement('li');
+  liRow.setAttribute('class', 'row');
+
+  var divForImg = document.createElement('div');
+  divForImg.setAttribute('class', 'column-half');
+  liRow.appendChild(divForImg);
+
+  var entryImg = document.createElement('img');
+  entryImg.setAttribute('alt', 'journal-image');
+  entryImg.setAttribute('src', data.photoUrl);
+  divForImg.appendChild(entryImg);
+
+  var divForSection = document.createElement('div');
+  divForSection.setAttribute('class', 'column-half right');
+  liRow.appendChild(divForSection);
+
+  var sectionForText = document.createElement('section');
+  divForSection.appendChild(sectionForText);
+
+  var headerSection = document.createElement('h2');
+  var h2TextContent = document.createTextNode(data.title);
+  headerSection.appendChild(h2TextContent);
+  sectionForText.appendChild(headerSection);
+
+  var textSection = document.createElement('p');
+  var pTextContent = document.createTextNode(data.notesInput);
+  textSection.appendChild(pTextContent);
+  sectionForText.appendChild(textSection);
+
+  return liRow;
+}
+
+$formEntry.addEventListener('submit', handleSubmit2);
+function handleSubmit2(event) {
```

Comment:
`2`? You don't need a separate `submit` event handler, you should just add this behavior to your existing `submit` event handler.